### PR TITLE
[MODIFY](client): clean up search page map, add marker {CT}

### PR DESCRIPTION
**Pull request by catharp**

### CODE REVIEW

File: server/handler.js [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+}
+
 module.exports.getPlaces = function(req, res) {
-  // Accepts a query with the following fields: 
-  // location, radius in meters (defaults to 500m), keyword (can be multiple keywords), minprice, maxprice, opennow
-  // location can be in the form of:
-    // - {lat, lng} 
-    // - {latitude, longitude}
-    // - {address} (so location.address is an address/partial address, in string form)
+  // Accepts a query with the following fields:
```
Comment (Hackmenate) [inline server/handler.js:15]: Oops, why is this comment going back to what it used to be? Rebasing error?

Comment (catharp) [reply server/handler.js:15]: Oh... yeah, that was my only conflict and I thought that was the new version...

File: server/utils/apicalls.js [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
-var dotenv = require('dotenv').config();
+var dotenv = require('dotenv').config();  // includes api keys
+
+// google maps js api loader
+module.exports.googleMapsLoader = function() {
+  let url = 'https://maps.googleapis.com/maps/api/js'
+
+  // build query string for api call
+  let qs = {
+    key: dotenv.GOOGLE_PLACE_API_KEY,
```
Comment (Hackmenate) [inline server/utils/apicalls.js]: GOOGLE_PLACE_API_KEY does not exist, we DO have GOOGLE_PLACE**S**_API_KEY, but wouldn't we want GOOGLE_MAPS_API_KEY? Could be the cause of the API key error.

**Comment by catharp** [comment]
all better :+1:

**Review by valerieernst — COMMENTED** [review]
In the Map.jsx file, you're handling state without Redux. I think everything that affects state is supposed to be done through actions and reducers.

**Review by zeninspire — APPROVED** [review]

**Comment by zeninspire** [comment]
to Val's comments: the state thing would work right now as it is being handled solely through React. Redux stuff is on the way, it's like treading on ice trying to change everything >_<

**Pull request closed without merging**